static/js/TopicLandingPage/TopicLandingSearch.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {GeneralAutocomplete} from "../GeneralAutocomplete";
+import Sefaria from "../sefaria/sefaria";
+import {SearchButton} from "../Misc";
+
+const getSuggestions = async (input) => {
+    if (input.length <=1) {
+      return [];
+    }
+    const word = input.trim();
+
+    const _getFormattedPath = (slug, lang) => {
+      const categories = Sefaria.topicTocCategories(slug);
+      if(!categories){return ""}
+      const titles = categories.map((cat) => cat[lang]);
+      return "("+ titles.join(` > `) + ")";
+    }
+
+    const parseSuggestions = (d, lang) => {
```

Comment:
use better parameter name than `d`.

Reply:
This function only uses `d[1]`. Maybe only pass that to begin with a give it a clear name.